modules/integration/registry/tests-deprecate/src/test/java/org/wso2/carbon/registry/smoke/test/GovernanceRegistrySmokeTestCase.java

```diff
@@ -387,7 +387,7 @@ private void smokeTestSubscriptions(String smokeTestCollection, String smokeTest
         InfoAdminServiceStub infoAdminServiceStub =
                 TestUtils.getInfoAdminServiceStub(loggedInSessionCookie);
         Assert.assertEquals(infoAdminServiceStub.subscribe(smokeTestCollection,
-                "https://localhost:9443/services/Hello", "CollectionUpdated",
+                "https://localhost:10343/services/Hello", "CollectionUpdated",
```

Comment:
Hi Amila,
Thanks for pointing this uot. Since these test were written using old test framework it does not have automation.xml. So I have removed these to a constant class. Please find the fix in PR #372 .
I have done the same change for other places as well.
Thanks